src/doc/dev_manual/LiveCustomerResponse.rst

```diff
@@ -1,244 +1,417 @@
     or it may be days. If it is days. Its conceivable the issue could be
-    handled in the following week by a new primary/backup pair.
-  * If customer indicates immediate response required, primary should inquire
-    the whole team to arrange another developer who can handle it.
-
-Relation to Programmatic Work
+    handled in the following week by a new **Primary/Backup** pair.
+  * If customer indicates immediate response is required, the **Primary**
+    should inquire the whole team to arrange another developer who can
+    handle it.
+
+Housekeeping and Low-hanging Fruit Type Issues
+----------------------------------------------
+
+Part of the reason for developing this process is the acknowledgment of the
+existence of a different category of work,
+`Software Reliability Engineering (SRE) <https://en.wikipedia.org/wiki/Site_Reliability_Engineering>`_,
+that is an essential part of maintaining the overall quality of a software
+product as well as the productivity of both developers and users of the
+software alike.
+
+Issues that impact one user's productivity often impact others. Likewise for
+developer productivity issues. Often, these kinds of issues can wind up falling
+through the cracks of traditional software project management and planning
+processes. However, such issues also often represent low cost high benefit
+improvements in quality of either the software itself or the development
+or deployment processes supporting it. We refer to issues of this nature
+as general *housekeeping* or *low-hanging fruit* type issues.
+
+Apart from acknowledging their existence, a key part of this process is the
+allocation of resources for the sole purpose of supporting
+**Customer Response** and developing a practice of continuously resolving
+general housekeeping or low-hanging fruit type issues arising from
+**Customer Response** inquiries.
+
+Consequently, another key role of the **Primary** is to use any time not working
+active inquiries to fix *low-hanging fruit* issues; either those the **Primary**
+is currently managing or those from the *backlog*. As a rule of thumb, low-hanging
+fruit is considered to be anything that the **Primary** believes is fixable
+within a half-day's (4 hours) worth of effort. When there are many such tasks in
+the system to work on, the **Primary** is free to use his/her judgment to decide
+which s/he can most productively address.
+
+Part of the acknowledgment of this new category of work is the new issue tracker
+for tracking it. *New* **Customer Response** activity will start with an issue 
+being added in this new issue tracker. However, there are likely a number of
+issues of this same kind already mixed in with our *normal* product development
+issues backlog. These should probably be audited for whether or not they are
+an issue of the *general housekeeping* or *low-hanging fruit* type here and
+then appropriately re-labeled.
+
+.. danger::
```

Comment:
This is a good idea. At the risk of sounding sarcastic, maybe this should be made into a ticket with issues divided among developers so that we can keep track of this and do it.

Reply:
Sure...Am awaiting other members of team to weigh in on this issue though.

Reply:
@visit-dav/visit-developers should we add a "housekeeping" or "low-hanging-fruit" label?

Reply:
> @visit-dav/visit-developers should we add a "housekeeping" or "low-hanging-fruit" label?

I will file a new issue for this